docs/qtile_docs/graph.py

```diff
@@ -58,54 +59,80 @@ def node_args(self, enabled=True, highlight=False, relative_url=str()):
         }
 
 
+def calc_vertex_position(number_nodes, include_origin=True, radius=2, offset=0, rounding=2):
+    """
+    Generator to calculate x, y coordinates for vertices on a regular polygon.
```

Comment:
can you make this comment `regular inscribed polygon`? i had to stare at your math for a while to understand what was happening.